hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/record/AbstractReplicatedRecordStore.java

```diff
@@ -86,7 +86,7 @@ private Object remove(InternalReplicatedMapStorage<K, V> storage, Object key) {
             storage.remove(marshalledKey, current);
         }
         if (replicatedMapConfig.isStatisticsEnabled()) {
-            getStats().incrementRemoves(Clock.currentTimeMillis() - time);
+            getStats().incrementRemoves(System.nanoTime() - time);
```

Comment:
nit: rename argument of `#incrementRemoves()` to `latencyNanos` and, optionally, rename the method itself to `#incrementRemovesLatencyNanos` (see `LocalMapStatsImpl` for example). The same consideration applies to other `#increment*()` methods.